### ISSUE TITLE
PhpBaseLexer calls non-existent method self.Channel and references SkipChannel

### ISSUE THREAD

**Issue by shimismith** [antlr/grammars-v4#1735]
In 
```python
 def PopModeOnCurlyBracketClose(self):
        if self._insideString:
            self._insideString = False
            self.channel(SkipChannel)
            self.popMode()
```
in PhpBaseLexer.py, `self.channel` does not exist and there is not access to `SkipChannel` in this file. I think the intended functionality should be `self._channel = PhpParser.PhpParser.SkipChannel` where `PhpParser` is the parser generated from the grammar. Please correct me if I'm mistaken.

Thanks

**Comment by KvanTTT** [comment]
I wasn't working on PHP grammar for Python target. Maybe @thierrymarianne can help?

**Comment by thierrymarianne** [comment]
As far as i recall, you might be correct with regards to `self._channel` (which should replace `self.channel`). However, I believe `SkipChannel` comes from `PhpLexer` (There is an example of a generated lexer  at this URL https://github.com/thierrymarianne/wild-goose-python-lexing-parsing-validator/blob/python3/PhpLexerParser/PhpLexer.py#L1042).

**Comment by shimismith** [comment]
Yes, my bad. `SkipChannel` is in `PhpLexer`. I got mixed up there. It doesn't appear to imported properly though.
